### ISSUE TITLE
Firefox: Text of toggle buttons not announced.

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#1626]
**Reported by kevinchao89 on 2011-06-29 20:04**
Firefox 7.0a1 (2011-06-29) and NVDA Snapshot 4511. 
1. Open 
   http://archive.dojotoolkit.org/nightly/dojotoolkit/dijit/tests/form/test_Button.html 
2. press h until you reach the heading that says "toggle buttons".
3. press B to move to the first button.
4. press the SPACEBAR or enter on the first button and NVDA should announce the new text on the button as it changes but it does not.

**Comment by leonardder** [comment]
CC @MichaelDCurran @JosephSL @JCSTeh

**Comment by jcsteh** [comment]
Still not fixed. Mozilla bug needs to be filed.

**Comment by jcsteh** [comment]
Firing name change events due to descendant changes is covered by [Mozilla bug 1422913](https://bugzilla.mozilla.org/show_bug.cgi?id=1422913). Name change events for other cases is covered by [Mozilla bug 493683](https://bugzilla.mozilla.org/show_bug.cgi?id=493683). I think the latter is most appropriate to this test case; I think it's using aria-labelledby. Both are difficult to fix without performance penalties and aren't a big problem in real world usage, which is why this hasn't been fixed yet.

**Comment by jcsteh** [comment]
Fixed in Firefox 87 (currently Nightly).
